introduction.md

```diff
@@ -1,2 +1,6 @@
 I am Subramanyam S, making commit to get to next challenge in Mastering Github by CodeSchool.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?